Public/Src/Utilities/Utilities/Tracing/CounterCollection.cs

```diff
@@ -349,8 +359,8 @@ static CounterCollection()
         /// Creates a collection with a counter for every value of <typeparamref name="TEnum"/>.
         /// Note that the enum should be dense, since this creates <c>MaxEnumValue - MinEnumValue + 1</c> counters.
         /// </summary>
-        public CounterCollection()
-            : base((ushort)s_counterTypes.Length)
+        public CounterCollection(CounterCollection<TEnum> parent = null)

```

Comment:
This PR doesn't have any users yet. Totally fine just checking in case you expected there to be.

Reply:
That's right. Currently there are no uses for this, but we do have plans for it in the future.